app/views/users/home.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class = "position-relative">
+  <h1 class = "text-center m-5 text-muted">Home Budget App</h1>
+  <div class="sign-btn-wrapper text-center d-flex flex-column">
+    <button class = 'btn btn-primary m-1'>
+      <%= link_to "LOG IN", new_user_session_path, id: "loginBtn" , class:"btn btn-primary" %>
+    </button>
+    <button class = 'btn btn-primary m-1'>
+      <%= link_to "SIGN UP", new_user_registration_path, id: "signupBtn", class:"btn btn-primary" %>
+    </button>
+  </div>
+</div>
```

Comment:
- [ ] Lets follow the requirements and add some styles to make your application look even better:

Your design  | Behance design
------------- | -------------
<img width="320" alt="Screenshot 2022-05-06 at 23 05 12" src="https://user-images.githubusercontent.com/77038610/167201897-f6db1e1a-e8ab-4b79-a279-279887653a7a.png">  | <img width="230" alt="Screenshot 2022-05-06 at 23 04 41" src="https://user-images.githubusercontent.com/77038610/167201836-c5824e5b-8d73-478f-b16e-a4684566a8c1.png"> 
| <img width="320" alt="Screenshot 2022-05-06 at 23 06 01" src="https://user-images.githubusercontent.com/77038610/167202002-f97ea260-dfb1-415f-a8bb-8e67d7f2069e.png"> | <img width="229" alt="Screenshot 2022-05-06 at 23 06 38" src="https://user-images.githubusercontent.com/77038610/167202091-e0bf2325-7d9c-42d3-be86-5feadc34fa3c.png">
 |  <img width="321" alt="Screenshot 2022-05-06 at 23 07 25" src="https://user-images.githubusercontent.com/77038610/167202207-4b887a39-a46d-4454-a59e-3a63e3cd3287.png"> | <img width="227" alt="Screenshot 2022-05-06 at 23 07 04" src="https://user-images.githubusercontent.com/77038610/167202155-8c6a5393-4db2-4e4f-b23f-3faef0b43448.png">

- [ ] Kindly update:
- * Typography: font  size and weight.
- * Layout: composition and space between elements